charts/manifests/templates/utils/README.md

```diff
@@ -91,6 +94,210 @@ String, YAML Structure
 {{- $values := include "bedag-lib.utils.common.mergedValues" (dict "type" "serviceAccount" "root" .) }}
 ```
 
+## [Configs](./_configs_.tpl)
+
+### Content
+---
+
+Renders Config content based on your needs
+
+#### Arguments
+
+If an as required marked argument is missing, the template engine will intentionally.
+
+  * `.config` - Config Structure [e.g. .config_files.application_properties ](Required)
+  * `.format` - Define the format when including the template. The format per config (`.config.format`) value takes precedence, if present.
+  * `.context` - Inherited Root Context (Required).
+
+
+#### Structure/Example
+
+This template supports the following key structure:
+
+
+**values.yaml**
+```
+config_files:
+
+  # Config Structure: Here we define a config file we want to render with a custom format
+  application_properties:
+
+    ## Format allows you to define your own pattern on how to format the key value pairs. 
+    ## You can use $.loop.key as placeholder for your key values and $.loop.value for the value placeholder.
+    ## This only works, when the content is kind map, otherwise it will be dumped to yaml without applying the format (Optional)
+    #
+    ## In the following example we want to use = ase key value indicator, therefor we use the following format:
+    format*: "{{ $.loop.key }}={{ tpl $.loop.value $ | quote }}"
+
+    ## For the format to apply, the content needs to be a map, otherwise it's just templated to yaml (Required)
+    content: 
+      spring.datasource.jdbcUrl: "jdbc:postgresql://postgresql:5342/postgres"
+      spring.datasource.driver-class-name: "org.postgresql.Driver"
+      spring.datasource.username: "${DB_USER}"
+      spring.datasource.password: "${DB_PASSWORD}"
+      api.auth.openid.authorizationUrlForSwagger: "{{ $.Values.config.properties.oauth.url }}"
+      
+  # Renders the given content just as yaml and allows templating
+  application_environment:
+    content: | 
+      some.environment = {
+        debug: false,
+        apiUrl: "{{ $.Values.config.properties.oauth.url }}",
+  
+        openIdConnect: {
+          redirectUri: window.location.origin + '/index.html',
+          requireHttps: false
+        }
+      }
+
+## Custom Config Structure we access from the configurations
```

Comment:
Custom Config Structure we (can?) access from the values/configuration (in the above example?).